packages/admin-panel/src/dataTables/useRuntimeParameters.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { useCallback } from 'react';
```

Comment:
Note: this is a new hook to separate runtime params state out from the rest of the state.